src/main/java/nc/recipe/vanilla/CraftingRecipeHandler.java

```diff
@@ -58,7 +58,7 @@ public static void registerCraftingRecipes() {
 		blockCompress(NCBlocks.block_depleted_thorium, "ingotThorium230");
 		blockCompress(NCBlocks.block_depleted_uranium, "ingotUranium238");
 		blockOpen(NCItems.americium, 3, NCBlocks.block_depleted_americium);
-		blockOpen(NCItems.californium, 2, NCBlocks.block_depleted_californium);
+		blockOpen(NCItems.californium, 1, NCBlocks.block_depleted_californium);
```

Comment:
This is because I reached the conclusion that the number for each line needs to be unique.